BMM.Core/ViewModels/CuratedPlaylistViewModel.cs

```diff
@@ -55,7 +55,6 @@ public Playlist CuratedPlaylist
         {
             _playlistManager = playlistManager;
             TrackInfoProvider = new AudiobookPodcastInfoProvider(TrackInfoProvider);
-            TextSource = new MvxLanguageBinder(GlobalConstants.GeneralNamespace, nameof(TrackCollectionViewModel));
```

Comment:
you can probably clean up the GlobalConstants since GeneralNamespace and some other entries are no longer needed